tourney/stats.py

```diff
@@ -186,9 +185,11 @@ def teams_key(pair):
 
   def general_response(self, lookup):
     top_amount = 5
+    top_range = range(top_amount)
     team_amount = 10
+    team_range = range(team_amount)
     total_dur = fmt_duration(self.__newest_score_time - self.__oldest_score_time)
-    return """
+    return """\
```

Comment:
Don't put this one. It's on purpose with the newline it yields. Same beneath at line 222.